### PR TITLE
updating installation documentation to python 3.7

### PR DESCRIPTION
**Pull request by acetousk**
Since this file was updated python 3.7 has come out, and is now standard. Therefore, making a couple simple changes makes it much easier to install. 
Making a code block and changing the numbers to Xs makes it more confusing rather than putting the latest numbers in and then saying if there is more recent version then use replace the numbers from this link.

### CODE REVIEW

File: v5/getting-started/linux.rst [hunk 1]
```diff
@@ -5,17 +5,17 @@ Installing on Linux
...
-2. Check the latest version of the PROS CLI on `our releases page <https://github.com/purduesigbots/pros-cli3/releases/latest>`_, and run :code:`python3.6 -m pip install --user https://github.com/purduesigbots/pros-cli/releases/download/3.X.X/pros_cli_v5-3.X.X-py3-none-any.whl`, replacing the number after 'python' with the version you installed and the Xs with the numbers you found before. If you wish to install for all users, run the command with :code:`sudo` and remove the :code:`--user` flag.
+1. If you do not already have one installed, install a version of Python greater than or equal to 3.6 (3.7 recommended)
+2. Check the latest version of the PROS CLI on `our releases page <https://github.com/purduesigbots/pros-cli3/releases/latest>`_, and run :code:`python3.7 -m pip install --user https://github.com/purduesigbots/pros-cli/releases/download/3.1.4/pros_cli_v5-3.1.4-py3-none-any.whl`, replacing the numbers after 'python' with the version you installed and the X.X.Xs with the numbers you found before. If you wish to install for all users, run the command with :code:`sudo` and remove the :code:`--user` flag.
```
Comment (HotelCalifornia) [inline v5/getting-started/linux.rst]: Why did you change the `X`s here?

File: v5/getting-started/linux.rst [hunk 2]
```diff
@@ -5,17 +5,17 @@ Installing on Linux
 Installing the toolchain
 ------------------------
 
-1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
+1. If you are using a Debian-based distribution of Linux of version >18.04, you can find the toolchain `here <https://www.ubuntuupdates.org/package/core/disco/universe/base/gcc-arm-none-eabi>`_.
```
Comment (HotelCalifornia) [inline v5/getting-started/linux.rst]: Is this true? I haven't heard anything about this. The version 18.04 here implies that you mean this works specifically on Ubuntu, rather than on any compatible Debian-derived distribution, so I'd like to verify that it works on those too.

File: v5/getting-started/linux.rst [hunk 3]
```diff
@@ -5,17 +5,17 @@ Installing on Linux
 Installing the toolchain
 ------------------------
 
-1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
+1. If you are using a Debian-based distribution of Linux of version >18.04, you can find the toolchain `here <https://www.ubuntuupdates.org/package/core/disco/universe/base/gcc-arm-none-eabi>`_.
 
-.. note:: If you are using a Debian-based distribution of Linux of version >18.04, you can find the toolchain `here <https://www.ubuntuupdates.org/package/core/disco/universe/base/gcc-arm-none-eabi>`_.
+.. note:: If you are using and older Debian-based distrobution of Linux the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
```
Comment (HotelCalifornia) [inline v5/getting-started/linux.rst]: "and" -> "an"
"distrobution" -> "distribution"
"...of Linux the instructions..." -> "...of Linux, follow the instructions..."

File: v5/getting-started/linux.rst (hunk 1 above)
Comment (acetousk) [reply v5/getting-started/linux.rst]: Changing the `X's` makes it so that you can copy and paste in a version that *will* work and run the command, because other wise it takes one additional unnecessary step.

File: v5/getting-started/linux.rst (hunk 2 above)
Comment (acetousk) [reply v5/getting-started/linux.rst]: I've actually only dealt with ubuntu deviations while installing pros i could try it on Debian and see if there are any errors with that.

File: v5/getting-started/linux.rst (hunk 3 above)
Comment (acetousk) [reply v5/getting-started/linux.rst]: That seems perfect.

**Comment by acetousk** [comment]
> I'm not opposed to adding Ubuntu-specific instructions, but I am wary of misrepresenting them as instructions that work for any Debian-based distribution, as there are some things that can be done on Ubuntu but not Debian.

File: v5/getting-started/linux.rst (hunk 1 above)
Comment (HotelCalifornia) [reply v5/getting-started/linux.rst]: Maybe, but now someone will have to remember to change this file whenever a new version is released.
